Scripts/Runtime/Entities/Job/FloodSetComponentJob.cs

```diff
@@ -29,7 +30,7 @@ public FloodSetComponentJob(ComponentTypeHandle<T> typeHandle, T value)
             m_Value = value;
         }
 
-        public unsafe void Execute(ArchetypeChunk chunk, int chunkIndex, int firstEntityIndex)
+        public unsafe void Execute(in ArchetypeChunk chunk, int unfilteredChunkIndex, bool useEnabledMask, in v128 chunkEnabledMask)
         {
             UnsafeCollectionUtil.FloodSetBuffer(chunk.GetComponentDataPtrRW(ref m_TypeHandle), 0, chunk.Count, m_Value);
```

Comment:
Because the Query is the input here I disagree. If I wanted to apply to disabled I'd specify that in the query.

If this were a low level job where you're passing in the chunk yourself then I'd agree but since it's operating on queries we should obey its request.